src/makeCalculator.js

```diff
@@ -38,6 +38,40 @@
  */
 function makeCalculator() {
   // write code here
+  const calculatorObject = {
+    result: 0,
+
+    add(param) {
+      calculatorObject.result += param;
+    },
+
+    subtract(param) {
+      calculatorObject.result -= param;
+    },
+
+    multiply(param) {
+      calculatorObject.result *= param;
+    },
+
+    divide(param) {
+      calculatorObject.result /= param;
```

Comment:
Consider implementing a check to avoid division by 0

Reply:
Ok, if param = 0, divide(parap) would return infinity? or some error?